gen/enum.go

```diff
@@ -66,7 +66,11 @@ func enum(g Generator, spec *compile.EnumSpec) error {
 		<$fmt := import "fmt">
 		<$json := import "encoding/json">
 		<$math := import "math">
-		<$strconv := import "strconv">
+		<if checkErrOnUnknownEnum ->
+			<$errors := import "errors">
+		<else ->
+			<$strconv := import "strconv">
+		<end ->
```

Comment:
We can just drop this and do the imports at the site where the package is needed.

```
  <$errors := import "errors">
  <$errors>.New(..)
```